sources/HUBViewController.m

```diff
@@ -1051,7 +1051,10 @@ - (CGPoint)overlayComponentCenterPoint
     CGRect frame = self.view.bounds;
     frame.origin.y = self.collectionView.contentInset.top;
     frame.size.height -= self.visibleKeyboardHeight + CGRectGetMinY(frame);
-    return CGPointMake(CGRectGetMidX(frame), CGRectGetMidY(frame));
+
+    CGPoint proposedCenterPoint = CGPointMake(CGRectGetMidX(frame), CGRectGetMidY(frame));
+    return [self.delegate centerPointForOverlayComponentInViewController:self
```

Comment:
If the delegate is `nil` this will return `CGPointZero`, which is not what we want :)